frontend/src/reviews/ReviewsList.tsx

```diff
@@ -1,36 +1,55 @@
 import { List } from '@mui/material'
 import { useGetReviewRequestsForUser } from 'actions/review'
-import { useEffect, useState } from 'react'
+import { useGetCurrentUser } from 'actions/user'
+import { useCallback, useEffect, useState } from 'react'
 import EmptyBlob from 'src/common/EmptyBlob'
 import Loading from 'src/common/Loading'
 import MessageAlert from 'src/MessageAlert'
 import ReviewItem from 'src/reviews/ReviewItem'
-import { ReviewRequestInterface } from 'types/interfaces'
+import { Decision, ReviewRequestInterface } from 'types/interfaces'
 
 type ReviewsListProps = {
-  isActive?: boolean
   kind?: 'release' | 'access' | 'all'
 }
 
-export default function ReviewsList({ isActive = true, kind = 'all' }: ReviewsListProps) {
-  const { reviews, isReviewsLoading, isReviewsError } = useGetReviewRequestsForUser(isActive)
+export default function ReviewsList({ kind = 'all' }: ReviewsListProps) {
+  const { reviews, isReviewsLoading, isReviewsError } = useGetReviewRequestsForUser()
+  const { currentUser, isCurrentUserLoading, isCurrentUserError } = useGetCurrentUser()
   const [filteredReviews, setFilteredReviews] = useState<ReviewRequestInterface[]>([])
 
+  const doesNotContainUserResponse = useCallback(
```

Comment:
This var name and the logic below doesn't match. The logic is checking that the user hasn't approved the review, but the var name suggests it should be looking to see if the user has responded at all.

I'm guessing the logic is correct and the name should probs be changed to something like `doesNotContainUserApproval`. Otherwise the logic will need updating to remove the `response.decision === Decision.Approve` check.

Reply:
Updated name